signer/plugin_test.go

```diff
@@ -13,6 +13,8 @@ import (
 	"time"
 
 	"github.com/notaryproject/notation-core-go/signature"
+	_ "github.com/notaryproject/notation-core-go/signature/cose"
+	_ "github.com/notaryproject/notation-core-go/signature/jws"
```

Comment:
Currently most of the tests in notation-go signer package are not executing because`signature.RegisteredEnvelopeTypes()` returns empty array. These lines fixes that issue